WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/TitleAndTextFieldRow.swift

```diff
@@ -19,6 +19,8 @@ struct TitleAndTextFieldRow: View {
     ///
     @Binding private var titleWidth: CGFloat?
 
+    @Binding private var contentColor: Color
```

Comment:
I'm thinking we can use a constant for this color instead of a `@Binding` property. The reason is that we don't need to change this color from within the component, only read it from the input.

The view will be re-rendered when its text changes, so with the input color from outside, its color will change appropriately. Does this make sense?

Reply:
Great suggestion, the `@Binding` ended up redundant, thanks! It's fixed here: https://github.com/woocommerce/woocommerce-ios/pull/7036/commits/f67b35312f8f5b6d4d0d6100a0d892f80afdb2cf